components/chart/search/index.tsx

```diff
@@ -38,12 +38,11 @@ export default function Search({
           currentVersion.nodes &&
           Object.keys(currentVersion.nodes).map((k) => {
             const m = currentVersion.nodes[k];
-
             return (
               'Node ' +
               m.id +
               ' > ' +
```

Comment:
Let's remove the word "Node" with the id, I think no need for them

Reply:
done